src/pal/src/file/file.cpp

```diff
@@ -1720,9 +1581,7 @@ GetFileAttributesA(
     struct stat stat_data;
     DWORD dwAttr = 0;
     DWORD dwLastError = 0;
-    CHAR * UnixFileName;
-    int length = 0;
-    PathCharString UnixFileNamePS;
+    PathCharString UnixFileName;
```

Comment:
Could you please change the UnixFileName casing to match other locals (start with lowercase)